src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/Elite2ApiService.java

```diff
@@ -28,15 +28,12 @@ public void putAttendance(final Long bookingId, final long activityId, final Eve
 
     public void putAttendanceForMultipleBookings(final Set<BookingActivity> bookingActivities, final EventOutcome eventOutcome) {
         final var url = "/bookings/activities/attendance";
-        final var performance = eventOutcome.getPerformance();
-        final var comments = eventOutcome.getOutcomeComment();
-
-        final var body = Map.of(
-                "bookingActivities", bookingActivities,
-                "eventOutcome", eventOutcome.getEventOutcome(),
-                "performance", performance != null ? performance : "",
```

Comment:
Sending an empty string caused the elite to throw a bad request because it couldn't find a performance value of '""'